.github/workflows/build_test_deploy.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - name: Setup node.js
         uses: actions/setup-node@v1
         with:
-          node-version: '12.x'
+          node-version: '18.x'
```

Comment:
Is it worth having a constant value so we can change it in one place, this may be overkill.

Reply:
Good shout. Have done this.